bangazon/bangazon_api/serializers.py

```diff
@@ -1,9 +1,47 @@
-from bangazon_api.models import Customer, ProductType, PaymentType, Order, Product
 from rest_framework import serializers
+from bangazon_api.models import PaymentType, Order, ProductType, Product, Customer
+
+
+class PaymentTypeSerializer(serializers.HyperlinkedModelSerializer):
+
+    """
+    Creates PaymentType Serializer
+    @rtwhitfield84
+
+    """
+
+    class Meta:
+        model = PaymentType
+        fields = ('customer', 'first_name', 'last_name', 'account', 'expiration_date', 'ccv')
+
+class ProductTypeSerializer(serializers.HyperlinkedModelSerializer):
+
+    """
+    Creates ProductType Serializer
+    @rtwhitfield84
+
+    """
+
+    class Meta:
+        model = ProductType
+        fields = ('category')
+
+class OrderSerializer(serializers.HyperlinkedModelSerializer):
+
+    """
+    Creates Order Serializer
+    @rtwhitfield84
+
+    """
+  
+    class Meta:
+      model = Order
+      fields = ('active', 'customer', 'payment_type')
+
 
 
 class ProductSerializer(serializers.HyperlinkedModelSerializer):
-  """
```

Comment:
Can we delete these three """ marks that are not closed? This may be my issue, but could you resolve it, please?